README.md

```diff
@@ -1,14 +1,59 @@
-# CS3219-AY22-23-Project-Skeleton
+# CS3219-AY22-23-S1-G13
 
-This is a template repository for CS3219 project.
+# Introduction 
+This project is an adaptation of the project PeerPrep that has been completed in the previous semesters of CS3219. This project provides a collaborative technical interview training platform, where one can receive a coding question, and collaborate with someone to answer the question. 
+
+
+# Running Project on Deployed Service
+
+The project is currently deployed [here](https://frontend-66acladbaq-uc.a.run.app). All APIs are also deployed, and able to be accessed through the frontend from the link. 

```

Comment:
Should we list out all the deployed urls of the different services here? Technically they can just try out the services in postman or something

Reply:
Perhaps we could put this in the report?